src/main/java/cloud/fogbow/ms/core/service/WhiteList.java

```diff
@@ -12,9 +14,13 @@
     private static final String SEPARATOR = ",";
 
     private List<String> membersList;
+    private List<String> authorizedTargetMembers;
+    private List<String> authorizedRequesterMembers;
     
-    public WhiteList() {
+    public WhiteList() throws ConfigurationErrorException {
```

Comment:
Just to make thinks more consistent, let's call this AllowList, instead of WhiteList.

Reply:
Done.